manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Ryujinx.Ryujinx.Ava
+PackageVersion: 1.1.1217
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles: []

```

Comment:
@russellbanks I don't know why it appears, but in the recent PRs opened on my account, "NestedInstallerFiles: []" appears.

Reply:
Thanks for reporting. This will be an easy fix :)

Reply:
Fixed in https://github.com/russellbanks/Komac/commit/d984c065d2a986adf1a56e5657cc3e7deb6e1cf1